Add  document-question-answering in task_summary (#23252)
huggingface/transformers: docs/source/en/task_summary.mdx

score: 0.9327, start: 30, end: 54, answer: huggingface/transformers
```

> ### Docuement Question answering

Reviewer: ```suggestion
### Document question answering
```

> Document Question answering is a task that answers natural language questions from a document. Unlike token-level question answering task which takes text as input, document question answering takes an image of a document as input along with a question about the document and returns an answer. Document Question answering can be used to parse structured documents and extract key information from documents. For example total amount and change amount can be extracted from a receipt.

Reviewer: ```suggestion
Document question answering is a task that answers natural language questions from a document. Unlike a token-level question answering task which takes text as input, document question answering takes an image of a document as input along with a question about the document and returns an answer. Document question answering can be used to parse structured documents and extract key information from documents. In the example below, the total amount and change due can be extracted from a receipt.
```

      'sequence': 'Hugging Face is a community-based open-source platform for machine learning.'}]
    ```

## Multimodal

> Multimodal tasks require a model to process multiple data modalities (text, image, audio, video) to solve a particular problem. A simple example of a multimodal task could be image captioning where the model takes an image as input and outputs a sequence of text describing the image or some properties of the image. Although multimodal models seem to work with different data types internally the preprocessing steps help the model convert all the data types into a similar form called embeddings (vectors or list of numbers that holds meaningful information about the data). So the model learns relationships between sequence of patches (small chunks of images) of image embeddings and text embeddings to complete a task like image captioning.

Reviewer: ```suggestion
Multimodal tasks require a model to process multiple data modalities (text, image, audio, video) to solve a particular problem. Image captioning is an example of a multimodal task where the model takes an image as input and outputs a sequence of text describing the image or some properties of the image. 

Although multimodal models work with different data types, internally, the preprocessing steps help the model convert all the data types into embeddings (vectors or list of numbers that holds meaningful information about the data). For a task like image captioning, the model learns relationships between image embeddings and text embeddings.
```
